back/src/graphql/type/user.js

```diff
@@ -105,16 +97,16 @@ export const register = {
       VALUES (?, ?, ?, ?)
     `;
     // TODO: Use bcrypt rather than plain text.
-    const userNumber = getUserNumber();
-    const [rows, _fields] = await conn.execute(SQL_ADD_USER, [
-      args.email,
-      args.password,
-      args.nickname,
-      // If this user is the first one, then he is the asmin.
-      userNumber === 0,
-    ]);
-    const res = (await getUserById(rows.insertId))[0];
-    return res;
+    return getUserNumber().then(async (userNumber) => {
```

Comment:
So `getUserNumber` is a `async` function right? I think use this code below will be a better idea (Because it will be more clear than callback. It is new syntax):
```
const userNumber = await getUserNumber();
```